tests/core/Observer.spec.ts

```diff
@@ -501,4 +504,81 @@ describe('Observer', async () => {
     expect(revertInvalidTransactionsSpy).toHaveBeenCalledTimes(0);
     expect(getFirstValidTransactionSpy).toHaveBeenCalledTimes(0);
   });
+
+  describe('waitUntilCountOfTransactionsUnderProcessingIsLessOrEqualTo', () => {
+    it('should wait until transactionsUnderProcessing is greater than count', async () => {
+      const originalDefaultTestTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+      jasmine.DEFAULT_TIMEOUT_INTERVAL = 4000; // it takes 3 seconds to remove all elements
```

Comment:
Is it possible avoid increase the total test run by 3+ seconds? Is this the magic jasmine timer can help with? Or maybe make the 1000 a private/constant property so you can mock?